src/main/java/lotto/domain/LottoNumber.java

```diff
@@ -0,0 +1,65 @@
+package lotto.domain;
+
+import java.util.*;
+
+public class LottoNumber {
+    private static final int MIN_LOTTO_NUMBER = 1;
+    private static final int MAX_LOTTO_NUMBER = 45;
+    private static final Map<Integer, LottoNumber> CACHE = new HashMap<>();
+
+    static {
+        for (int i = MIN_LOTTO_NUMBER; i <= MAX_LOTTO_NUMBER ; i++) {
+            CACHE.put(i, new LottoNumber(i));
+        }
+    }
```

Comment:
아래와 같이 변경해볼 수 있을 것 같아요. 😉

```suggestion
    private static final Map<Integer, LottoNumber> CACHE;

    static {
        CACHE = IntStream.rangeClosed(MIN_LOTTO_NUMBER, MAX_LOTTO_NUMBER)
            .boxed()
            .collect(Collectors.toMap(Function.identity(), LottoNumber::new));
    }
```

`Function.identity()`가 조금 낯설 수 있는데 추가 설명을 드리자면 자기 자신을 반환하는 람다식이에요! 🙂

<img width="487" alt="image" src="https://github.com/next-step/java-lotto-kakao/assets/47850258/584d3e81-a99e-4ec1-83c5-1a22d224b741">